File: The-Golden-Leaf-server/src/main/java/com/example/datban/repository/DatBanRepository.java
package com.example.datban.repository;

import com.example.datban.model.DatBan;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface DatBanRepository extends JpaRepository<DatBan, Long> {
}

File: app/src/main/java/com/example/giaodien/data/repository/DatBanRepository.kt
package com.example.giaodien.data.repository

import com.example.giaodien.data.model.DatBan
import com.example.giaodien.data.network.RetrofitInstance

class DatBanRepository {

    suspend fun datBan(datBan: DatBan): DatBan {
        return RetrofitInstance.api.createDatBan(datBan)
    }
}
